lib/apiservers/engine/backends/image.go

```diff
@@ -234,6 +234,8 @@ func imageConfigToDockerImageInspect(imageConfig *metadata.ImageConfig, productN
 	return inspectData
 }
 
+const DefaultDockerRegistry = "registry-1.docker.io/"
```

Comment:
is this a stable url? i worry about locking all VIC to this is docker hub does some sort of dynamic allocating.